diff --git a/node_modules/capacitor-firebase-auth/ios/Plugin/Plugin/Plugin.swift b/node_modules/capacitor-firebase-auth/ios/Plugin/Plugin/Plugin.swift
index 6121a23..3a02197 100644
--- a/node_modules/capacitor-firebase-auth/ios/Plugin/Plugin/Plugin.swift
+++ b/node_modules/capacitor-firebase-auth/ios/Plugin/Plugin/Plugin.swift
@@ -175,7 +175,7 @@ public class CapacitorFirebaseAuth: CAPPlugin {
             call.success()
         } catch let signOutError as NSError {
             print ("Error signing out: %@", signOutError)
-            call.reject("Error signing out: %@", signOutError)
+            call.reject("Error signing out: %@", nil, signOutError)
         }
     }
 }
